lib/interface/cli/commands/hybrid/init.cmd.js

```diff
@@ -425,10 +426,11 @@ const initCmd = new Command({
             const answers = await inquirer.prompt(questions);
             kubeContextName = kubeContextName || answers.context;
             kubeNamespace = kubeNamespace || answers.namespace;
+            appProxyHost = appProxyHost || answers.appProxyHost;
             shouldMakeDefaultRe = _.isUndefined(shouldMakeDefaultRe) ? answers.shouldMakeDefaultRe : shouldMakeDefaultRe;
             shouldExecutePipeline = _.isUndefined(shouldExecutePipeline) ? answers.shouldExecutePipeline : shouldExecutePipeline;
             // eslint-disable-next-line object-curly-newline
-            ({ httpProxy, httpsProxy, noProxy, appProxyHost } = answers);
```

Comment:
this is the bug. it would just get override if it's given by flag instead of question